package.json

```diff
@@ -48,16 +48,17 @@
     "pre-commit": "lint-staged && npm run lint",
     "build:templates": "gulp templates",
     "start": "npm run build:templates && webpack-dev-server --open --mode development --config config/webpack.dev.mjs",
-    "test": "karma start karma.config.js",
+    "lint": "eslint .",
+    "test:unit": "npm run lint && karma start karma.config.js",
     "watch:templates": "gulp watch",
     "build": "npm run build:templates && webpack --mode production --config config/webpack.prod.mjs",
     "start:prod-local": "cd dist && http-server -p 4200 -c-1 -b",
     "prettier": "prettier --write \"**/*.{js,json,css,scss,less,md,ts,html}\"",
-    "lint": "eslint .",
-    "e2e": "playwright test",
-    "e2e:debug": "playwright test --project chromium --headed",
+    "test:e2e": "npm run lint && playwright test",
+    "test:e2e:debug": "playwright test --project chromium --headed",
+    "test:e2e-report": " npx playwright show-report",
```

Comment:
Should this be `test:e2e:report` for consistency?

Also, there is an unneeded leading whitespace in the command.

Reply:
Fixed